Neos.Neos/Migrations/Code/Version20230801154834.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Neos\Flow\Core\Migrations;
+
+
+/**
+ * Replace defaultUriSuffix configuration with uriPathSuffix in default site configuration
+ */
+class Version20230801154834 extends AbstractMigration
+{
+
+    public function getIdentifier(): string
+    {
+        return 'Neos.Neos-20230801154834';
+    }
+
+    public function up(): void
+    {
+        $this->moveSettingsPaths(['Neos', 'Flow', 'mvc', 'routes', 'Neos.Neos', 'variables', 'defaultUriSuffix'], ['Neos', 'Neos', 'sites', '*', 'uriPathSuffix']);
```

Comment:
wow thanks, i see now why we need https://github.com/neos/flow-development-collection/pull/3125 ^^ ❤️  thanks for taking care!